### PR TITLE
Added plugin code, tests

### PR DESCRIPTION
**Pull request by kunagpal**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,10 @@
   ],
   "author": "Postman Labs <help@getpostman.com> (=)",
   "license": "Apache-2.0",
+  "dependencies": {
+    "lodash": "4.17.4",
+    "postman-collection": "3.0.0"
```
Comment (shamasis) [inline package.json]: Instead of depending on collection, can we make be passed over? Then move this to Dev dependencies.

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+        notAcceptable: 406,
+        rateLimited: 429
+    };
+
+/**
+ * Adds postman-collection assertions via the chai-postman plugin interface.
+ *
+ * @param {Object} chai - A Chai assertion wrapper class provided by the chai library.
+ */
+module.exports = function (chai) {
```
Comment (shamasis) [inline lib/index.js]: Put this inside another closure and accept SDK instance. 

```
module.exports = function (sdk) {
  return function (chai) {
    // …
  };
}
```

File: package.json (hunk 1 above)
Comment (kunagpal) [reply package.json]: Addressed.

File: lib/index.js (hunk 2 above)
Comment (kunagpal) [reply lib/index.js]: Addressed.

**Pull request merged**